src/tactic/ui/tools/repo_browser_wdg.py

```diff
@@ -2158,25 +2156,42 @@ def get_dir_context_menu(my, mode="strict"):
 
 
         if search_type:
+            ingest_key = "repo_browser_ingest:%s" % search_type
+            ingest_settings = jsonloads(WidgetSettings.get_value_by_key(ingest_key))
+            ingest_data_view = ingest_settings.get("ingest_data_view")
```

Comment:
if ingest_data_view is not specified.. would it be "" or "None" in the js later.. same with ingest_custom_view.. can u verify?